src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -747,13 +747,15 @@ export default class DefaultAudioVideoController
     // Check in case this function has been called before peer connection is set up
     // since that is necessary to try to update remote videos without the full resubscribe path
     needsRenegotiation ||= this.meetingSessionContext.peer === undefined;
-    // If updating remote video without negotiation fails, fall back to renegotiation
+    // If updating local or remote video without negotiation fails, fall back to renegotiation
     needsRenegotiation ||= !this.updateRemoteVideosFromLastVideosToReceive();
+    needsRenegotiation ||= !this.updateLocalVideoFromPolicy();
     // `MeetingSessionContext.lastVideosToReceive` needs to be updated regardless
     this.meetingSessionContext.lastVideosToReceive = this.meetingSessionContext.videosToReceive;
     if (!needsRenegotiation) {
       this.logger.info('Update request does not require resubscribe');
-      return true; // Skip the subscribe!
+      this.actionFinishUpdating();
```

Comment:
Can you log why we need this?

Reply:
Added comment: update encoding in updateLocalVideoFromPolicy and apply the new bitrate to send here

Reply:
Maybe can be more explicit that we are reusing the function.
```
// Call `actionFinishUpdating` to apply the new encoding parameters that may have been set in `updateLocalVideoFromPolicy`.
```